src/serializer/ResidualHeapSerializer.js

```diff
@@ -980,7 +980,11 @@ export class ResidualHeapSerializer {
 
   _serializeValueSymbol(val: SymbolValue): BabelNodeExpression {
     let args = [];
-    if (val.$Description) args.push(t.stringLiteral(val.$Description));
+    if (val.$Description) {
```

Comment:
Align with code in visitor: Either always do `instanceof Value` check, or always do `val.$Description !== undefined`. Avoid doing indiscriminate truthiness-check, which is a bit funky in JavaScript.